WordPress/src/main/java/org/wordpress/android/ui/posts/editor/media/EditorMedia.kt

```diff
@@ -299,14 +300,7 @@ class EditorMedia @Inject constructor(
     }
 
     fun onMediaUploadPaused(listener: EditorMediaUploadListener, media: MediaModel, error: MediaError) = launch {
-        val properties: Map<String, Any?> = withContext(bgDispatcher) {
-            analyticsUtilsWrapper
-                .getMediaProperties(media.isVideo, null, media.filePath)
-                .also {
-                    it["error_type"] = error.type.name
-                }
-        }
-        analyticsTrackerWrapper.track(EDITOR_UPLOAD_MEDIA_FAILED, properties)
+        analyticsTrackerWrapper.track(EDITOR_UPLOAD_MEDIA_PAUSED)
```

Comment:
What is the rationale for diverging from the error event and _not tracking_ `properties`? My presumption is that the properties provide _some_ value.

Reply:
> What is the rationale for diverging from the error event and not tracking properties? My presumption is that the properties provide some value.

Good point. The initial rationale was that `properties` would relate more specifically to errors. After logging out the key/values pairs of `properties`, this is what would be sent to the paused analytics event:

```
ext: 
age_ms: 2526548679
bytes: 231174
mime: image/jpeg
megapixels: 0
error_type: CONNECTION_ERROR
```

This matches what is sent to `EDITOR_UPLOAD_MEDIA_FAILED`, naturally, so it's probably worth sending to the paused event too. I've updated `onMediaUploadPaused` to send `properties`. 👍